bugbot/rules/component.py

```diff
@@ -89,6 +89,55 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if bug_data.get("class") == "Fenix"
+            and bug_data["prob"][bug_data["index"]]
+            >= self.get_config("general_confidence_threshold")
+        ]
+
+        def get_confidence_threshold(
+            bug_data, general_confidence_threshold, confidence_threshold
+        ):
+            if bug_data["class"] == "General":
+                return general_confidence_threshold
+            return confidence_threshold
```

Comment:
We could simplify this by changing the function to check whether it passes the threshold or not.

Selecting a good name for the function will be important to improve the readability.

Reply:
Added a new function called `meets_threshold()` that checks if a classification meets the specified threshold: bc68d06132884591efb2653977567238bd0ea681

Reply:
I think it will be cleaner to select the threshold inside `meets_threshold()`. All that we need to pass is `bug_data`.

Reply:
Done here: ff5b63363ce48bf29f2ba730efd91ff7de42a0d5